.github/workflows/scripts/build.sh

```diff
@@ -1,23 +1,22 @@
 #!/bin/bash
 set -eux
 
-python_executable=python$1
```

Comment:
Definitely "python_executable=python$1" is a cryptic way to interface the desirable python flavor (not ideal), but why we limit the script to python3 ?

Reply:
What other options are there in our ROCm release docker (or anywhere)?

Reply:
python2 ?

Reply:
This code will not be within the top 1000 things that will break :)